AUTHORS

```diff
@@ -1,3 +1,6 @@
 Timur Shemsedinov <timur.shemsedinov@gmail.com>
+Dimon Durak <Dimon Durak>
+Artem Chernenkiy <notthewhite@gmail.com>
 Alexey Orlenko <eaglexrlnk@gmail.com>
 Mykola Bilochub <nbelochub@gmail.com>
+Vlad <Dzyubavlad@gmail.com>
```

Comment:
@DzyubSpirit I think we prefer real names unless you have strong reasons to not be called Vlad Dzyuba. I was asking about the correct spelling of your last name, actually :)
What about email, maybe I should have been more explicit too: the question is whether it should be written with a capital letter or I can lowercase it.